talibs/Makefile

```diff
@@ -0,0 +1,21 @@
+export V?=0
+
+# If _HOST or _TA specific compilers are not specified, then use CROSS_COMPILE
+HOST_CROSS_COMPILE ?= $(CROSS_COMPILE)
```

Comment:
In `optee_test/Makefile` it's `CROSS_COMPILE_HOST` instead. I think we should stick with one order.

Reply:
Agreed, but I have just copied what other examples do so this should be addressed separately IMO.